Source/Scene/Model.js

```diff
@@ -3088,12 +3098,13 @@ define([
      * @exception {RuntimeError} Failed to load external reference.
      */
     Model.prototype.update = function(frameState) {
-        if (frameState.mode !== SceneMode.SCENE3D) {
+        if (frameState.mode === SceneMode.MORPHING) {
             return;
         }
 
         var context = frameState.context;
         this._defaultTexture = context.defaultTexture;
+        this._projection = frameState.mapProjection;
```

Comment:
Why is this a property?  Can't we just pass it to `updateNodeHierarchyModelMatrix`?